### DISCUSSION TITLE
Return Error Instantly when try to Download

### DISCUSSION THREAD

**Discussion by NelsonCat** [aclap-dev/video-downloadhelper#676] [Help and Q&A]
### Your question

After click the download video, it return error instantly.

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://missav.com/sone-037-chinese-subtitle",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [in#0 @ 000001994a26b600] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://surrit.com/dae1714a-6e84-448c-945f-0c8777aa6368/720p/video.m3u8.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "id": "coapp_failure_13f9466b-1e38-423a-a447-569b809c1a10",
    "downloadable_id": "downloadable_4570039129165663"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [in#0 @ 000001994a26b600] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://surrit.com/dae1714a-6e84-448c-945f-0c8777aa6368/720p/video.m3u8.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "id": "coapp_failure_13f9466b-1e38-423a-a447-569b809c1a10",
    "downloadable_id": "downloadable_4570039129165663"
  }
}

**Comment by GrampaWildWilly** [comment]
Problems with missav have already been reported here.  Read this: #519.  Be sure to click any links you see that say, "Show x previous replies."
